### PR TITLE
TIKO/TASK2

### PR DESCRIPTION
**Pull request by TigranIskandaryan324**

### CODE REVIEW

File: src/com/egs/tik/task2/ex2/MyUser.java [hunk 1]
```diff
@@ -0,0 +1,140 @@
+package com.egs.tik.task2.ex2;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class MyUser implements Serializable{
```
Comment (serghovhannisyan) [inline src/com/egs/tik/task2/ex2/MyUser.java:7]: why serializeable ?

File: src/com/egs/tik/task2/ex2/MyUser.java [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    public static ArrayList<String> users = new ArrayList<>();
+
+
+    //MENU METHOD
+    public static void functionName(String s){
```
Comment (serghovhannisyan) [inline src/com/egs/tik/task2/ex2/MyUser.java:12]: refactor code

File: src/com/egs/tik/task2/ex2/MyUser.java [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+            case "add":{
+                String values = inputNameAge();
+                add(values);
+                System.out.println("Input function name ( add , remove , list , exit)");
+                break;
+            }
+            case "remove":{
+                System.out.println("Input remove index");
+                int index = removeIndex();
+                myRemove(index);
```
Comment (serghovhannisyan) [inline src/com/egs/tik/task2/ex2/MyUser.java:23]: use proper method names. removeUser ?

File: src/com/egs/tik/task2/ex2/MyUser.java [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+        else {
+            System.out.println("List is ampty or index out of bands");
+        }
+    }
+
+
+
+    //INPUT REMOVE INDEX METHOD ////
+    public  static int removeIndex(){
+        Scanner scanner = new Scanner(System.in);
```
Comment (serghovhannisyan) [inline src/com/egs/tik/task2/ex2/MyUser.java:83]: why init scanner each time? either pass scanner as a parameter or declare it as a field or something like that

File: src/com/egs/tik/task2/ex2/MyUser.java (hunk 2 above)
Comment (movsisyanh) [reply src/com/egs/tik/task2/ex2/MyUser.java:12]: :D

**Pull request closed without merging**